code/modules/research/techweb/all_nodes.dm

```diff
@@ -539,6 +539,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/rld
+	id = "rld"
+	display_name = "Advanced Sanitation Technology"
+	description = "Clean things better, faster, stronger, and harder!"
```

Comment:
Update these to be different from the janitor node

Reply:
fuck forgot that